src/PlaywrightSharp.Firefox/FirefoxPage.cs

```diff
@@ -259,7 +259,7 @@ public Task SetEmulateMediaAsync(MediaType? mediaType, ColorScheme? colorScheme)
 
         public Task<byte[]> GetPdfAsync(string file, PdfOptions options) => throw new NotImplementedException();
 
-        public Task<IPage> GetOpenerAsync() => throw new NotImplementedException();
+        public async Task<IPage> GetOpenerAsync() => await _openerResolver().ConfigureAwait(false);

```

Comment:
Why do we need the async/await?

Reply:
`_openerResolver` returns `Task<Page>` and we need to return as `IPage`